src/main/java/com/google/sps/servlets/DataFromDatabase.java

```diff
@@ -0,0 +1,159 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.ByteArray;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/data-from-db")
+public class DataFromDatabase extends HttpServlet {
+
+  private static final String TABLE_SELECT_PARAM = "table-select";
```

Comment:
I like the use of these variables to make the code simpler!

Reply:
Thanks!